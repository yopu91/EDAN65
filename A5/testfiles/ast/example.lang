int a(){
a=b;
return a;
}
