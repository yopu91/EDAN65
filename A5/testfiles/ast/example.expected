Program
  List
    ParamFunc
      IntType
      IdDecl
      List
      List
        Assignment
          IdUse
          IdUse
        ReturnStmt
          IdUse
